packages/client/src/components/SendTokens/components/AssetSelector.js

```diff
@@ -70,11 +71,10 @@ const AssetSelector = () => {
         <NFTSelector
           nftsToDisplay={nftsToDisplay}
           selectedNFT={selectedNFT}
-          setSelectedNFT={(selected, url) => {
+          setSelectedNFT={(selected) => {
```

Comment:
why do we remove this `url`?

Reply:
Good question! Previously `selected` is a concatenated string  looks like `A.f8d6e0586b0a20c7.ZeedzINO.Collection-0`, url is the thumbnail url. I changed selected to be the nft object, which contains the url, so we can use the nft name and nft id on the TransactionDetails component